bitcoin.md

```diff
@@ -18,27 +18,37 @@ curl --proto '=https' --tlsv1.2 -sSf https://raw.githubusercontent.com/stacksfou
 Since we'll be importing a wallet into bitcoin, it's **highly recommended** that Bitcoin is compiled locally. \
 That said, to run a pre-compiled binary of `bitcoind`, you can download and install the binary using these commands:
 
-```
-$ export BTC_VERSION=22.0
-$ sudo curl -L https://bitcoin.org/bin/bitcoin-core-${BTC_VERSION}/bitcoin-${BTC_VERSION}-x86_64-linux-gnu.tar.gz -o /tmp/bitcoin-22.0.tar.gz
+```bash
+$ export BTC_VERSION=25.0
+$ sudo curl -L https://bitcoin.org/bin/bitcoin-core-${BTC_VERSION}/bitcoin-${BTC_VERSION}-x86_64-linux-gnu.tar.gz -o /tmp/bitcoin-${BTC_VERSION}.tar.gz
 $ sudo tar -xzvf /tmp/bitcoin-${BTC_VERSION}.tar.gz -C /tmp
 $ sudo cp /tmp/bitcoin-${BTC_VERSION}/bin/* /usr/local/bin/
 ```
 
 ## Source Install
 
-```
-$ export BTC_VERSION=22.0
+```bash
+$ export BTC_VERSION=25.0
 $ git clone --depth 1 --branch v${BTC_VERSION} https://github.com/bitcoin/bitcoin /tmp/bitcoin && cd /tmp/bitcoin
-$ sh contrib/install_db4.sh .
+$ make -C depends NO_BOOST=1 NO_LIBEVENT=1 NO_QT=1 NO_SQLITE=1 NO_NATPMP=1 NO_UPNP=1 NO_ZMQ=1 NO_USDT=1
 $ ./autogen.sh
-$ export BDB_PREFIX="/tmp/bitcoin/db4" && ./configure BDB_LIBS="-L${BDB_PREFIX}/lib -ldb_cxx-4.8" BDB_CFLAGS="-I${BDB_PREFIX}/include" \
- --disable-gui-tests \
- --enable-static \
- --without-miniupnpc \
- --with-pic \
- --enable-cxx \
- --with-boost-libdir=/usr/lib/x86_64-linux-gnu
```

Comment:
i would be surprised if you don't need this. will test.

Reply:
Worked without

Reply:
confirmed, this seems fine and `libminiupnpc` is a default package now